core/controllers/classifier_test.py

```diff
@@ -132,13 +132,43 @@ def test_error_on_different_signatures(self):
             '/ml/trainedclassifierhandler', self.payload,
             expect_errors=True, expected_status_int=401)
 
-    def test_error_on_invalid_message(self):
+    def test_error_on_invalid_job_id(self):
```

Comment:
I think the method name should still include some info about the fact that this is an invalid job id in the message.

Reply:
Updated method name